tools.yaml

```diff
@@ -34,3 +34,321 @@ batman.r-forge.r-project.org:
+    url: https://creativecommons.org/licenses/by-nc/4.0/
+
+metaboanalyst.ca:
+  name: MetaboAnalyst
+  description: |
+    "Tool suite for metabolomics analysis"
+  url: https://www.metaboanalyst.ca
+  author: 
+    - name: Xia Lab
+      email: jeff.xia@mcgill.ca
+  license: 
+    kind: GNU General Public License
+    url: http://www.gnu.org/licenses/
+  
+github.com/dgrapov/MetaMapR:
+  name: MetaMapR
+  description: |
+    "Local or web-based tool for generating biochemical interaction (KEGG), structural similarity 
+    (PubChem Substructure Fingerprints), mass spectral similarity and correlation based networks."
+  url: http://dgrapov.github.io/MetaMapR/
+  author: 
+    - name: Dmitry Grapov
+      email: dgrapov@gmail.com
+  license: 
+    kind: GNU General Public License
+    url: https://github.com/dgrapov/MetaMapR/blob/master/LICENSE
+
+metap.helmholtz-muenchen.de/metap2:
+  name: metaP-server
+  description: |
+    "automates data analysis for the processing of metabolomics experiments."
+  url: http://metap.helmholtz-muenchen.de/metap2/
+  author: 
+    - name: Gabi Kastenmüller
+      email: presse@helmholtz-muenchen.de
+    - name: Werner Römisch-Margl
+  license:
+    kind: 
+    url:  https://www.pathway.jp/licensing/commercial.html
+
+metexplore.fr:
+  name: MetaExplore
+  description: |
+    "MetExplore is a free academic service allowing; Importing/storing/sharing genome scale meta­bolic networks; Map­ping poly­omics data; 
+    Path­way enrichment; Visu­al­iz­ing networks; Min­ing net­works based on data and net­work structure; and Com­put­ing fluxes"
+  url: http://www.metexplore.fr
+  author: 
+    - name: Institut national de la recherche agronomique
+      email: "?"
+  license:
+    kind: Creative Commons Attribution-NonCommercial 4.0 International License.
+    url: https://creativecommons.org/licenses/by-nc/4.0/
+
+metscape.ncibi.org/: 
+  name: MetScape 3 for Cytoscape
+  description: |
+    "The MetScape Plugin for Cytoscape provides a bioinformatics framework for the visualization and interpretation of metabolomic 
+    and expression profiling data in the context of human metabolism."
+  url: http://metscape.ncibi.org/
+  author:
+    - name: Bill Duren
+      email: metscape-help@umich.edu
+    - name: Terry E. Weymouth
+    - name: Tim Hull
+    - name: Alla Karnovsky
+  license:
+    kind: "?"
+    url: "?"
+
+# Paid service 
+umetrics.com/products/simca: 
+  name: SIMCA
+  description: |
+    "Multivariate statistical analysis (PCA, PLSDA etc.)"
+  url: https://umetrics.com/products/simca
+  author: 
+    - name: Sartorius Stedim Biotech
+      email: umetrics@sartorius-stedim.com
+  license: 
+    kind:
+    url:
+
+# paid service
+statsoft.com/Products/STATISTICA-Features:
+  name: Statistica product features
+  description: |
+    "tools for statistical analysis of data"
+  url: http://www.statsoft.com/Products/STATISTICA-Features
+  author:
+    - name: 
```

Comment:
Omit

Reply:
Remove the entire entry?

Reply:
I meant omit the blank `-name:` property and really any blank properties.

Reply:
should I omit author: too?